src/Uno.UI/UI/Xaml/Shapes/Shape.layout.cs

```diff
@@ -338,7 +338,9 @@ private protected Size ArrangeAbsoluteShape(Size finalSize, NativePath? path, Fi
 		{
 			if (path! == null!)
 			{
+#if !__WASM__
```

Comment:
I would instead support `null` in the `Render` method on WASM

Reply:
On Wasm, `path` is the actual `Shape` instance (ie, `this`). It's never null actually.

And also we don't pass `path`.